automates/text_reading/src/main/resources/org/clulab/aske_automates/grammars/entities/grammar/entities.yml

```diff
@@ -20,13 +22,13 @@ rules:
     pattern: |
       [tag=/VBN|JJ/]
 
-
-  - name: single_capital_VP
-    label: Concept
-    priority: 1
-    type: token
-    pattern: |
-      [chunk='B-VP' & word = /[A-Z]/]
+  # note: error? (repetition) 
```

Comment:
@alicekwak a quick question: what was this a repetition of you thought? I can't find it. Thanks!

Reply:
Oops, I'm not sure why, but I had one more rule that is exactly the same as single_capital_VP rule. So I marked one of them off and made a note saying that the rule is a repetition. However, it seems like there's no repetition here, so I think we should enable the rule again. Sorry for the confusion!

Reply:
@alicekwak, thanks! No worries!